atomic_reactor/plugins/build_source_container.py

```diff
@@ -37,9 +37,10 @@ def export_image(self, image_output_dir: Path) -> Dict[str, Union[str, int]]:
         source_img = 'oci:{}'.format(image_output_dir)
         dest_img = 'docker-archive:{}'.format(output_path)
         cmd += [source_img, dest_img]
+        cleanup_cmd = ['rm', output_path]
 
         try:
-            retries.run_cmd(cmd)
+            retries.run_cmd(cmd, cleanup_cmd)
```

Comment:
It would have been a bit more flexible to have any callable as the cleanup action

e.g. like this
```python
retries.run_cmd(cmd, cleanup=lambda: output_path.unlink(missing_ok=True)
```
But I don't mind doing the cleanup by calling a subprocess